src/styles/globals.css

```
@@ -116,6 +117,15 @@
   }
   body {
     @apply bg-background text-foreground antialiased;
+    font-family: var(--font-sans);
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)